odfuzz/config.py

```diff
@@ -28,6 +28,7 @@ def __init__(self):
         self._urls_per_property = int(env_url_per_property)
         env_ignore_restriction = os.getenv(ENV_IGNORE_METADATA_RESTRICTIONS,DEFAULT_IGNORE_METADATA_RESTRICTIONS)
         self._ignore_restriction = env_ignore_restriction
+        self._http_method_enabled = "GET"
```

Comment:
OK, this is result of several discussion and limiting of the scope of PR to DirectBuilder only.

It will not work correctly with CLI usage of odfuzz, but that's is for another PR, probably after major refactoring anyway.